src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -867,3 +875,40 @@ export default class ClinicalData extends React.Component<
         return this.tabUI.component;
     }
 }
+
+export function toBoxPlotTable(data: any[], labels: string[]) {
+    const headers = labels.length === 1
+        ? <th colSpan={2}>{labels[0]}</th>
+        : [<th/>, labels.map(label => <th colSpan={1}>{label}</th>)]
+    return (
+        <table className="table table-striped">
```

Comment:
can you put a min-width of 400px on this table?  simple way of addressing issue where table doesn't fill the wrapper
![image](https://user-images.githubusercontent.com/186521/167882522-f98ef411-ed92-42e8-ae8d-b1d6efed5cf0.png)

Reply:
Fixed in [46954e1](https://github.com/cBioPortal/cbioportal-frontend/pull/4248/commits/46954e1507ddaa1b488cb9455a51e5a6d01e1385)